common/g_gametype.cpp

```diff
@@ -659,10 +653,7 @@ void G_LivesCheckEndGame()
 			if (tv.size() == 1)
 			{
 				GiveTeamWins(tv.front()->Team, 1);
-				SV_BroadcastPrintf(
-				    PRINT_HIGH,
-				    "%s team wins for having the highest score with %s left!\n",
-				    tv.front()->ColorString.c_str(), teams);
+				SV_BroadcastPrintf("%s team wins for having the highest score with %s left!\n", tv.front()->ColorString.c_str(), teams);
```

Comment:
Please run `clang-format` on this document.

`g_gametype` is a file I created recently, and if I was the one that created the file, I run `clang-format` on it using the settings at the root of the Odamex tree.  In other files, I try to selectively `clang-format` just the parts of the code I commit.